### ISSUE TITLE
Two instances of Sublime Text being launched from `Open in Editor`

### ISSUE THREAD

**Issue by fcgoogle** [desktop/desktop#8584]
### Describe the bug
使用编辑器打开的时候会打开两个编辑器界面，
我使用的默认编辑器是Sublime，当前仓库只有一个的情况下会打开两个一模一样的编辑器界面，内容相同；
如果有两个仓库在本地，当点击Open in sublime后会分别打开每个仓库的内容

>When you open it with an editor, two editor interfaces open.
<
>The default editor I use is Sublime. When there is only one current repository, two identical editor interfaces will be opened with the same content.
If there are two repositories locally, when you click Open in sublime, the contents of each repository will be opened separately.

### Version & OS
vesion 2.2.2 OS windows 10
Open 'About GitHub Desktop' menu to see the Desktop version. Also include what operating system you are using.

### Steps to reproduce the behavior

1. Go to 'the menu'
2. Click on 'Repository'
3. Scroll down to 'Open in sublime text'

### Expected behavior

only open the current Repository's readme.md or other files

### Actual behavior
Open multiple Repository's readme

### Screenshots
![image](https://user-images.githubusercontent.com/24488212/68230855-e47e4000-0034-11ea-9767-d34c7300e834.png)

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

**Comment by steveward** [comment]
Thanks for the report @fcgoogle. I tested this out on my Windows machine with Sublime Text and was unable to reproduce the issue.

Is it possible you clicked the `Open in Editor` button multiple times? This is a known issue in GitHub Desktop -- see https://github.com/desktop/desktop/issues/6864 for more information. There's a [comment](https://github.com/desktop/desktop/issues/6864) that apparently solved this for Sublime Text, so you could try that workaround to see if it helps.

**Comment by JarnoVgr** [comment]
Haven't been using Sublime Text recently. But the first window being opened is most likely because Sublime re-opens files that weren't saved on exit (called 'hot exit').

Untested work-around is to disable hot exit support via the Sublime Text config:
```
"hot_exit": false,
"remember_open_files": false,
```

tl;dr: Most likely not a problem with GitHub Desktop.

**Comment by outofambit** [comment]
@JarnoVgr thanks for updating us! Glad its working now!

**Comment by Eugene8571** [comment]
it is May 2020, have this issue

**Comment by steveward** [comment]
@Eugene8571 could you see if [this workaround](https://github.com/desktop/desktop/issues/6864) resolves the issue for you?

**Comment by Eugene8571** [comment]
somewhat.
with 'hot_exit": false, and "remember_open_files": false, it no more double
window and no previous file, if i run Sublime separately.
i get 1 window from git and prev file if run Sublime with
"remember_open_files": false. which is strange.

чт, 14 мая 2020 г. в 19:15, Steve Ward <notifications@github.com>:
